examples/experimental/openvino/ssd_mobilenet_v1_fpn/main.py

```diff
@@ -15,8 +15,8 @@
 
 import numpy as np
 import openvino.runtime as ov
-from openvino.tools.pot.api.samples.object_detection.data_loader import COCOLoader
-from openvino.tools.pot.api.samples.object_detection.metric import MAP
+from utils import MAP
+from utils import COCOLoader
```

Comment:
What do you think about removing this example?

Reply:
@andrey-churkin, do we need this example?
I don't mind about removing it.

Reply:
It was removed here https://github.com/openvinotoolkit/nncf/pull/2392

Reply:
Well, someone could have warned about this earlier. Thanks.